qemu/tests/expose_host_mtu.py

```diff
@@ -56,13 +56,14 @@ def cleanup_ovs_ports(netdst, ports):
 
         if is_ovs_backend(netdst) is True:
             ports = set(get_ovs_ports(netdst).splitlines()) - \
-                    set(ports.splitlines())
+                set(ports.splitlines())
             for p in ports:
                 process.system("ovs-vsctl del-port %s %s" % (netdst, p))
 
     netdst = params.get("netdst", "switch")
     if netdst in utils_net.Bridge().list_br():
-        host_hw_interface = utils_net.Bridge().list_iface()[0]
+        host_hw_interface = \
+            utils_net.Bridge().get_structure()[netdst]['iface'][0]
     elif is_ovs_backend(netdst) is True:
         host_hw_interface = get_ovs_ports(netdst)
         tmp_ports = re.findall(r"t[0-9]-[a-zA-Z0-9]{6}", host_hw_interface)
```

Comment:
Please help to update this, since I saw that `tmp_ports` maybe in another format.
```
Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.jumbo.q35.
Error reason: Command 'ifconfig t17-kDiI5M mtu 9000' failed.
stdout: b''
stderr: b'SIOCSIFMTU: No such device\n'
additional_info: None.
```
Keyword: `t17-kDiI5M`

Reply:
I'd like to fix this in the other PR https://github.com/autotest/tp-qemu/pull/1624. Because this bug is to fix bridge issue and thanks for point this out.

Reply:
Sure. We can open other one to fix all tmp_ports